resources/js/app.js

```diff
@@ -1,2 +1,17 @@
 require('./bootstrap');
 require('alpinejs');
+
+
+const closeAlertButton = document.getElementById("closeAlertButton");
```

Comment:
Please use AlpineJS to hide the element on click, rather than adding custom Javascript. I'm actually not sure how this would work either way, as the body isn't loaded yet by now and  this element will never exist.